index.js

```diff
@@ -0,0 +1,269 @@
+'use strict';
+let _ = require('lodash')
+  , redis = require('redis')
+  , modelQueue = false
+  , schemas = {};
+
+class ModelQueue {
+  constructor(config, logger) {
+    this.config = config;
+    this.config.messageKey = this.config.messageKey || 'eriksen-metadata';
```

Comment:
Maybe we could call this the `queueStateKey`, `queueTransitionKey`, or to keep the pubsub thing, the `pubsubKey` ... it took me a while to realize/remember that `message` in this case meant "pub sub message"